src/rofi_rbw/cache.py

```diff
@@ -0,0 +1,36 @@
+from collections import defaultdict
+from typing import List
+
+from .entry import Entry
+from .paths import cache_file
+
+
+class Cache:
+    def __init__(self):
+        cache_file.parent.mkdir(exist_ok=True)
+        cache_file.touch(exist_ok=True)
```

Comment:
Do we need to create the file first or can we use `open("w+")`?

Reply:
The first use of this file is a read, so this can't use `w+`. And if the file doesn't exist, a `FileNotFoundException` is raised.

Reply:
I'll resolve this discussion to get a more clear status of the PR, but don't hesitate to re-open if you disagree